TC/iocBoot/iocTC-IOC-01/st-common.lua

```diff
@@ -1,36 +1,20 @@
 package.path = package.path .. ';' .. os.getenv("UTILITIES") .. '/lua/luaUtils.lua;'
 ibex_utils = require "luaUtils"
 
--- Check a given file exists on disk. 
-function exists(file)
-	local f=io.open(file,"r")
-    if f~=nil then io.close(f) return true else return false end
- end
  
 -- Main function called by st-common.cmd
 function twincat_stcommon_main()
 	local motor_port = "L0"
 	local pv_prefix = ibex_utils.getMacroValue{macro="MYPVPREFIX"}
-	local tpy_file = ibex_utils.getMacroValue{macro="TPY_FILE"}
 	local ioc_name = ibex_utils.getMacroValue{macro="IOCNAME"}
 	local plc_version = ibex_utils.getMacroValue{macro="PLC_VERSION", default="1"}
-	-- Set TcIoc's scan rate to 150ms for ADS and 300ms for EPICS - this is to avoid ADS fields not updating in channel access quickly enough. 
-	iocsh.tcSetScanRate(150, 2)
-
-	local full_tpy_path = ibex_utils.getMacroValue{macro="TWINCATCONFIG"} .. "/" .. tpy_file
-	if not exists(full_tpy_path) then
-		print("invalid TPY file given: " .. full_tpy_path)
-		iocsh.exit()
-	end
+	local ads_port = ibex_utils.getMacroValue{macro="ADS_PORT", default="852"}
```

Comment:
do we need "default" here as i believe it is already applied at the st.cmd level?